sdk/release_notes.md

```diff
@@ -2,7 +2,4 @@
 <!-- Please add your release notes in the following format:
 - My change description (#PR/#issue)
 -->
-- Error handling for in-proc SDK usage inside isolated app. (#824)
```

Comment:
We are yet to release the non preview version of SDK with these changes. Can get this info from source control when we do that. In my PR,  am bumping up the package version again to preview version.